package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rsksmart/rif-relay-client",
-  "version": "1.0.0-alpha.0",
+  "version": "1.0.0-alpha.1",
```

Comment:
I wouldn't change the package version here, since we're going to change it before releasing it.

Reply:
I don't know. I find it a convenient way to create a draft PR before making any changes, so that I can move a task from to-do to in-progress.

Reply:
but tbh, I don't understand what you mean since you increased the version in your suggestion.

Reply:
changed